### ISSUE TITLE
Responsive Layout for Docs

### ISSUE THREAD

**Issue by tparisi** [tparisi/glam#50]
The documentation CSS needs to be reworked to use a responsive layout. I want this with simple media queries and maybe some Bootstrap. No Angular and nothing else fancy and preferably no build steps!

**Comment by ailadson** [comment]
i might as well do this too.

**Comment by tparisi** [comment]
Yes please! I merged your PR and first-pass it looks good but I need to test more...
